snippets/csharp/pipelines/Pipe.cs

```diff
@@ -53,7 +53,7 @@ async Task FillPipeAsync(Socket socket, PipeWriter writer)
             }
 
             // Tell the PipeReader that there's no more data coming.
-            writer.Complete();
+            await writer.CompleteAsync();
```

Comment:
The comment mentions a reader, but code completes a writer. Is that correct?

Reply:
Good catch

Reply:
I think the comment is fine as is. The call to CompleteAsync is "telling" the other side, the PipeReader, "that there's no more data coming."

Reply:
> I think the comment is fine as is. The call to CompleteAsync is "telling" the other side, the PipeReader, "that there's no more data coming."

I've thought that's the case. Is that fine to make it more explicit:
```
// By completing PipeWriter, we tell the PipeReader that there's no more data coming.
```

Reply:
Sounds good to me. Open a PR.

Reply:
Sounds good to me. You should open a PR.